keyboards/lazydesigners/dimple/keymaps/via/rules.mk

```diff
@@ -7,4 +7,3 @@ MIDI_ENABLE = no
 BLUETOOTH_ENABLE = no
 KEY_LOCK_ENABLE = no
 TERMINAL_ENABLE = no
```

Comment:
```suggestion
```

These + MIDI, Faux Clicky and Leader are disabled by default.

Reply:
Removed